src/components/gl-wrapper/__tests__/gl-wrapper-test-cases.js

```diff
@@ -9,4 +10,22 @@ testCases.basic = {
   element: <Basic />
 };
 
+noRenderCases.ie = {
+  description: "reason='insufficient ECMAScript 6 support'",
+  element: (
+    <GLWrapper reason="insufficient ECMAScript 6 support">
+      Your browser supports Mapbox GL!
+    </GLWrapper>
+  )
+};
+
+noRenderCases.webgl = {
+  description: "reason='insufficient WebGL support'",
+  element: (
+    <GLWrapper reason="insufficient WebGL support">
+      Your browser supports Mapbox GL!
+    </GLWrapper>
+  )
+};
+
```

Comment:
📝 This `noRenderCases` generate a snapshot so we can track if there are any changes the next time we update mapbox-gl-supported.